packages/core/src/Site/index.js

```diff
@@ -523,6 +526,8 @@ class Site {
         : filteredPage;
     });
     this.addressablePages = Object.values(filteredPages);
+    this.addressablePagesSource = Object.values(filteredPages).map(page =>
```

Comment:
> I don't think I've moved the existing variables around. Let me know if I misinterpreted your message!

hmm sorry, might've misread the earlier code

this looks good but to make it work with live reload you may need to maintain the same reference since we're only passing it to `pageConfig` once (clear and repush the array instead of `addressablePagesSource  = ...`)

Reply:
Ah, can I check if this would apply to `addressablePages` as well? 

Because in line 528, there's also the reassignment. Will `addressablePages` still work with live reload?

Reply:
yup, its not being passed anywhere else if I recall